### PR TITLE
feat(nextjs): support project structure under src folder

### PR DESCRIPTION
**Pull request by subzero10**
## Status
**READY**

## Description
Closes: #1129.

## TODO
- [x] Update script to check for `src` folder
- [x] Tests

### CODE REVIEW

File: packages/nextjs/src/copy-config-files.test.ts [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import fs from 'fs'
+import path from 'path'
+import { copyConfigFiles } from './copy-config-files';
+describe('copy-config-files', () => {
+
+  afterEach(() => {
+    mock.restore()
+  })
+
+  it('should copy config files to a project with pages router', () => {
```
Comment (BethanyBerkowitz) [inline packages/nextjs/src/copy-config-files.test.ts:11]: Thanks for adding tests!

Can we default to using `async/await` instead of `.then()`?

**Review by BethanyBerkowitz — DISMISSED** [review]
Nice! Minor comment, don't need to re-review.

**Comment by BethanyBerkowitz** [comment]
@subzero10 Do we need to continue with unit test for node 14? Looks like 14 is no longer supported: https://github.com/nodejs/release#release-schedule

File: packages/nextjs/src/copy-config-files.test.ts (hunk 1 above)
Comment (subzero10) [reply packages/nextjs/src/copy-config-files.test.ts]: 👍 Done!

**Comment by subzero10** [comment]
> @subzero10 Do we need to continue with unit test for node 14? Looks like 14 is no longer supported: https://github.com/nodejs/release#release-schedule

That'a good question. Seems like we can stop supporting Node 14, though we'd have to do a major version bump right? 🤔 Because as soon as we remove from CI, then we won't know when some piece of code would break on Node 14.

**Pull request merged**